.gitpod.yml

```diff
@@ -0,0 +1,21 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: cargo build --features kvm,virtualbox,xen
+
+vscode:
+  extensions:
+    - editorconfig.editorconfig
```

Comment:
Can you put a comment on these extensions and why they are needed ?

Reply:
Are you sure you want me to do that? Seems a bit redundant provided most of my comments would probably be paraphrasing the first sentence in the corresponding plugin summary. That said I have no problem with adding comments, just wanted to double check.

Reply:
The thing is that replaced the default ide for vscode, because rust analyzer support issues.
So at least a comment could be placed here to explain why we are using experimental extensions.

That's all i meant.

Reply:
Oh yeah, that makes sense. :D Good call 👍